src/main/js/components/dropdowns/jumplists.js

```diff
@@ -115,6 +55,29 @@ function generateDropdowns() {
       ),
   );
 
+  behaviorShim.specify(
+    ".dropdown-indicator",
+    "-clickable-dropdown-",
+    1000,
+    (element) =>
+      Utils.generateDropdown(
+        element,
+        createDropdownContent(
+          element,
+          element.getAttribute("data-model"),
+          element.getAttribute("data-children"),
+          element.getAttribute("data-href"),
+        ),
+        false,
+        {
```

Comment:
I would expect escape to close the jumplist and it doesn't currently, (left arrow does, but its not very intuitive)

Reply:
We can enhance the a11y support in the future I agree, but ATM I do not want to implement custom event handling and have better keyboard. ...but I actually think that is an epic since there are a lot of places and we further would want to analyse the current overall situation to then create tickets.